### PR TITLE
Resources branch

### PR DESCRIPTION
**Pull request by selynna**
Contains new folder called "resources" with md file about adding line notes

### CODE REVIEW

**Comment by zachlatta** [comment]
Any ideas on a more descriptive name than "resources"? Also – does it make sense to have resources in the root of the Hack Camp repo, instead of in the playbook folder? What if it was directly part of `ACTIVITIES.md`.

**Comment by selynna** [comment]
I'm not too sure it's considered an activity... We could put it in `CONTRIBUTING.md` because I feel like that may be a better alternative. Thoughts?

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -1431,3 +1432,21 @@ Want to send a link to everyone? _Slack._
...
+concepts of programming I (Jonathan) have not found anything clearer than
+CodeHS at explaining these basics.
+
+#### Cons:
+
+- the student nevers have the opportunity to build your own projects in CodeHS.
+  You are told everything that you are supposed to build.
+- perhaps even worse, you have no experience even in building _any_ kind
+  projects because you are always following a specific directions that keep
+  giving you the next step.
```
Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Is this supposed to be part of the playbook?

File: resources/resources.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Resources
+
+## Line Notes
+
+This indicates where there may be a typo or bug in the text, and is very helpful to the leaders as it shows them exactly where they have gone wrong.  There are three steps to achieve this:
+
+1. Using Github's "blame" function
```
Comment (zachlatta) [inline resources/resources.md]: GitHub generally has a capital H in their name.

File: resources/resources.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Resources
+
+## Line Notes
+
+This indicates where there may be a typo or bug in the text, and is very helpful to the leaders as it shows them exactly where they have gone wrong.  There are three steps to achieve this:
```
Comment (zachlatta) [inline resources/resources.md]: Can you wrap this at 80 characters? We also generally only use one space between sentences.

File: resources/resources.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+1. Using Github's "blame" function
+2. Looking for the appropriate commit.
+3. Adding the actual line note.
+
+### Using Github's "blame" function
+
+![Screenshot of Git Blame](/blame.png)
+
+In the screenshot above, this is where you can access git blame.  This is the first step to adding line notes.
```
Comment (zachlatta) [inline resources/resources.md]: Same thing as https://github.com/hackedu/hack-camp/pull/519/files#r37701718.

File: resources/resources.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+### Looking for the appropriate commit and adding line note:
+
+Let's say that there is a mistake in the workshop on line 34.  You would navigate to line 34's respective commit, like so, and add the line note by clicking the blue plus:
```
Comment (zachlatta) [inline resources/resources.md]: Same thing as https://github.com/hackedu/hack-camp/pull/519/files#r37701718.

File: cohort_4/playbook/ACTIVITIES.md (hunk 1 above)
Comment (selynna) [inline cohort_4/playbook/ACTIVITIES.md]: We used CodeHS for Cohort 2

@jonleung

Comment (zachlatta) [inline cohort_4/playbook/ACTIVITIES.md]: Perhaps this could be included in a separate pull request? It seems out of
scope of this one.

On Sat, Aug 22, 2015 at 6:40 PM, Selynna Sun notifications@github.com
wrote:

> In cohort_4/playbook/ACTIVITIES.md
> https://github.com/hackedu/hack-camp/pull/519#discussion_r37701723:
> 
> > +
> > +### CodeHS
> > +
> > +#### Pros:
> > +
> > +- is very good at introducing and meticulously explaining the basic
> > +concepts of programming I (Jonathan) have not found anything clearer than
> > +CodeHS at explaining these basics.
> > +
> > +#### Cons:
> > +
> > +- the student nevers have the opportunity to build your own projects in CodeHS.
> > -  You are told everything that you are supposed to build.
> >   +- perhaps even worse, you have no experience even in building _any_ kind
> > -  projects because you are always following a specific directions that keep
> > -  giving you the next step.
> 
> We used CodeHS for Cohort 2
> 
> @jonleung https://github.com/jonleung
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/519/files#r37701723.

## 

Zach Latta
hackEDU | zach@hackedu.us

Comment (jonleung) [inline cohort_4/playbook/ACTIVITIES.md]: @ssun098, I agree, @ssun098 can you take out the entire CodeHS section in this pull request? Thanks!

**Comment by selynna** [comment]
help @zachlatta i may have screwed something up

File: cohort_4/playbook/ACTIVITIES.md [hunk 6]
```diff
@@ -104,6 +104,7 @@
     - [Announcing to the Whole Group](#announcing-to-the-whole-group)
     - [Sharing Announcements Individually](#sharing-announcements-individually)
     - [Slack Announcements](#slack-announcements)
+- [Additional Resources](#additional-resources)
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md]: @ssun098 This is what is causing the tests to fail. There doesn't seem to be an `Additional Resources` heading.

Comment (selynna) [inline cohort_4/playbook/ACTIVITIES.md]: Oh god sorry that was the part for codehs and I deleted it. I'll fix that

Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md]: No worries! Once you get that in the PR, I'm happy to merge :)

**Comment by maxwofford** [comment]
@ssun098 I noticed there were a few merge commits in this PR. When I need to update a branch to things that occur on `master` I use the command `git rebase -i master` when the PR branch is checked out. 

https://www.atlassian.com/git/tutorials/merging-vs-rebasing

**Comment by selynna** [comment]
@MaxWofford help please ;_; I'm going to screw something up

**Comment by selynna** [comment]
Rebasing has never gone well for me

**Comment by maxwofford** [comment]
@ssun098 Don't worry about rebasing right now, that's just a tip for the future. As long as the `Additional Resources` section is deleted, I can merge this for you :)

**Comment by selynna** [comment]
@MaxWofford i think it works?

**Comment by maxwofford** [comment]
@ssun098 Looks good!
screenshot 2015-08-25 at 8 21 59 pm

**Comment by selynna** [comment]
@MaxWofford I'm going to assign you to merge

**Comment by maxwofford** [comment]
Will merge when tests finish.

**Comment by zachlatta** [comment]
@MaxWofford can we actually have a bit more conversation about the changes in this?

I think it'd make the most sense to put this in `ACTIVITIES.md` or `README.md` (or even create a `CONTRIBUTING.md` in the `playbook` directory). I'm very partial against creating new directories to accommodate for something like this.

**Comment by maxwofford** [comment]
@zachlatta Perhaps this could be included in the master `CONTRIBUTING.md`? I think that not many contributors are using the tools that are included in this PR

**Comment by zachlatta** [comment]
Can you make an issue to track that?

On Tue, Aug 25, 2015 at 11:27 PM, Max Wofford notifications@github.com
wrote:

> @zachlatta https://github.com/zachlatta Perhaps this could be included
> in the master CONTRIBUTING.md? I think that not many contributors are
> using the tools that are included in this PR
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/hackedu/hack-camp/pull/519#issuecomment-134855535.

## 

Zach Latta
hackEDU | zach@hackedu.us

**Comment by jonleung** [comment]
Hey! I unsubscribed from this issue. Please tag me again if you want to ping me with anything. Thanks!

**Comment by maxwofford** [comment]
Hmm, on second thought i'd like to hold off on adding this to the main `CONTRIBUTING.md` until we've formalized how we want club members to give input. We'll do our best to revisit this change when having that conversation.
